package-lock.json

```diff
@@ -0,0 +1,12233 @@
+{
```

Comment:
We recommend not including this file in a library as it will not be used when the library is installed.